pkg/util/initiator.go

```diff
@@ -264,14 +264,13 @@ func execWithTimeoutRetry(cmdLine []string, timeout, retry int) (err error) {
 
 func (nvmf *initiatorNVMf) Connect() (string, error) {
 	klog.Info("connections", nvmf.connections)
-	lossTmo := strconv.Itoa(CtrlLossTmo)
 	if len(nvmf.connections) == 1 {
-		lossTmo = nvmf.ctrlLossTmo
+		CtrlLossTmo *= 15
```

Comment:
we are updating a global variable here and using the same in connection string. The next time when this function is called, we'll continue to use 15 mins even when nvmf.connections != 1. Is this expected?

Reply:
Yes as the connection will be to the same Sb cluster which is either HA or non-HA